identity-and-access-management/iam/how-to/accept-invitation-to-orga.mdx

```diff
@@ -21,20 +21,47 @@ When you [create a Scaleway account](/console/my-account/how-to/create-an-accoun
+
+    <Message type="important">
+      If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAM Manager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.
+    </Message>
 
-    Always ensure that you are in the desired Organization at any time by checking the Organization name that displays in the top right corner of your console. In the image below, we are currently in the Marie Curie Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization, for example below we could switch to the Pierre Curie Organization.
+    You are redirected to the overview page of the Organization to which you were invited.
+
+    As an [IAM user](/identity-and-access-management/iam/concepts/#user) of an Organization, you must always ensure that you are in the desired Organization by checking the Organization name that displays in the top right corner of your console.
+
+    In the image below, we are currently in the Marion Créange Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization, for example below we could switch to the Documentation Team Organization.
 
     <Lightbox src="scaleway-orga-dropdown.webp" alt="" />
 
-    <Message type="note">
-      If you receive an invitation to join an Organization and you do not already have a [Scaleway account](/console/my-account/how-to/create-an-account/), you will be prompted to create one.
+## If you do not have a Scaleway account
+
+1. Click **Create your account** from your email invitation. The Scaleway console opens in a browser window.
+2. Create a Scaleway account. Follow the [How to create a Scaleway account](/console/my-account/how-to/create-an-account) documentation page to do so. Your Organization overview page displays.
+3. Click **Accept** in the top right pop-up.
+    <Lightbox src="scaleway-accept-orga-invite.webp" alt="" />
+
+    <Message type="important">
+      If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAM Manager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.
     </Message>
 
+    You are redirected to the overview page of the Organization to which you were invited.
+
+    As an [IAM user](/identity-and-access-management/iam/concepts/#user) of an Organization, you must always ensure that you are in the desired Organization by checking the Organization name that displays in the top right corner of your console.
+
+    In the image below, we are currently in the Marion Créange Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization, for example below we could switch to the Documentation Team Organization.
```

Comment:
Use the drop-down menu to navigate to a different Organization. For example, below, we could switch to the Documentation Team Organization.

Reply:
```suggestion
    In the image below, we are currently in the Marion Créange Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization. For example, below, we could switch to the Documentation Team Organization.
```